invenio_rdm_records/fixtures/users.py

```diff
@@ -72,6 +73,9 @@ def create(self, entry):
                 action = current_access.actions[action]
                 db.session.add(ActionUsers.allow(action, user_id=user.id))
 
+            if confirmed:
+                confirm_user(user)
```

Comment:
Will this actually work without an `allow` action? Since the `db.session.commit()` unsure if the `confirm user` will query for something that is not yet added to the session.

Reply:
I tried and yes, it works without the any allow action.